src/main/java/bungae/thunder/cakey/message/repository/MessageJpaRepository.java

```diff
@@ -1,8 +1,16 @@
 package bungae.thunder.cakey.message.repository;
 
 import bungae.thunder.cakey.message.domain.Message;
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface MessageJpaRepository extends JpaRepository<Message, Long> {}
+public interface MessageJpaRepository extends JpaRepository<Message, Long> {
+    @Query("SELECT m FROM Message m JOIN FETCH m.cake WHERE m.cake.id = :cakeId")
+    List<Message> findAllByCakeId(Long cakeId);
+
+    @Query("SELECT m FROM Message m JOIN FETCH m.sender WHERE m.sender.id = :senderId")
+    List<Message> findAllBySenderId(Long senderId);
```

Comment:
We can use default jpa method good

Reply:
https://velog.io/@ohzzi/Data-Jpa-findByXXXId-%EB%8A%94-%EB%B6%88%ED%95%84%EC%9A%94%ED%95%9C-join%EC%9D%84-%EC%9C%A0%EB%B0%9C%ED%95%9C%EB%8B%A4